calabash-cucumber/spec/integration/launcher/inject_dylibs_spec.rb

```diff
@@ -1,36 +0,0 @@
-
```

Comment:
This change should be in a separate change set.

You can push this change directly to /develop.